### ISSUE TITLE
Dependency glitch

### ISSUE THREAD

**Issue by SlimeDog** [MegaNarwhal/BiomeFinder#11]
Per [Rocologo](https://github.com/Rocologo/MobHunting/issues/603):
```
My guess is that the developer of Biomfinder has imported the class org.inventivetalent.update.spiget.UpdateCallback directly into the source code instead of adding it as a dependency in the pom.xml file. The dev should import Inventive talents source code into his own.

MobHunting is NOT dependend on Biomfinder, but it IS dependend on org.inventivetalent.update.spiget.UpdateCallback
```

**Comment by MegaNarwhal** [comment]
Should be fixed in bbab7bdd141751e5875e0b67cc46ef910e27da25.

**Comment by SlimeDog** [comment]
Confirmed fixed in BF 1.3.0. Thanks.
